02-KernelModules_CharacterDevice/Makefile

```
@@ -0,0 +1,11 @@
+KDIR := /home/eduard/Kernel_i386
+
+obj-m := Character_driver.o
+
+.PHONY: all clean
+
+all:
+	$(MAKE) -C ${KDIR} M=$$PWD modules
+
+clean:
+	$(MAKE) -C ${KDIR} M=$$PWD clean
```

```suggestion
	$(MAKE) -C ${KDIR} M=$$PWD clean
	
```

Please add empty line at the end of file